### DISCUSSION TITLE
Lost on what the hell to do

### DISCUSSION THREAD

**Discussion by JKareck123** [UltimaHoarder/UltimaScraper#1586] [Q&A]
So. Finally got the Json sorted and seemed like it's all ready to go. Problem is that whenever I load the start_ofd.py , it appears and then posts the "Site:0 = Onlyfans" and no matter what I do next, it ends up not working. Attached a screenshot of the script running because I'm so lost. Anyone know what I've done wrong?

https://i.imgur.com/jReHXxN.png

**Comment by PsychoSid** [comment]
What happens when you press '0' at this prompt to scrape OnlyFans as it says ?

**Comment by JKareck123** [comment]
I press '0' and this happens. Get a value rror at the end
![py_aUJzGheXU6](https://user-images.githubusercontent.com/88726079/129743380-afa0df70-bf1a-41fa-aed8-620d1f4abc51.png)

**Reply by vacuran** [reply]
It looks like it's having issues with your `auth.json`, I think that's where that `user_auth_filepath` ultimately comes from. Have you followed the directions for updating that properly?
